qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/RunTest.java

```diff
@@ -55,6 +56,7 @@
 		public static final String SELENIUM_VITALS = SELENIUM + " and "+ VITALS;
 		public static final String SELENIUM_CLINICAL_VISIT = SELENIUM + " and "+ CLINICAL_VISIT;
 		public static final String SELENIUM_ENCOUNTER = SELENIUM + " and "+ INPATIENT;
+		public static final String SELENIUM_PATIENT = SELENIUM + " and "+ PATIENT;
```

Comment:
```suggestion
		public static final String SELENIUM_PATIENT = SELENIUM + " and " + PATIENT;
```

Reply:
@kdaud I think there's some inconsistency in this file. What would be the best way to concatenate these variables. So many do not have that space before the + sign. Are we planning to fix them for consistency?

Reply:
> Are we planning to fix them for consistency?

would be a good deal if you fix them all.

Reply:
> > Are we planning to fix them for consistency?
> 
> would be a good deal if you fix them all.

@kdaud would that be ok to fix them in this PR or I create a ticket for that?

Reply:
fix them in this PR.

Reply:
@kdaud  I have done the fix. Anymore reviews?